build_targets/families.py

```diff
@@ -137,10 +137,12 @@ def source(self) -> List[str]:
             'emu.c',
             'cmu.c',
             'systick.c',
+            'gpio.c',
         )
 
     def c_flags(self) -> List[str]:
         return [
+            '-g',
```

Comment:
You probably didn't mean to commit this, right?

Reply:
yup, leftovers, I should add this to a if block with the debug option